torch/_torch_docs.py

```diff
@@ -469,6 +469,48 @@ def merge_dicts(*dicts):
     True
 """)
 
+add_docstr(torch.as_strided,
+           r"""
+as_strided(input, size, stride, storage_offset=0) -> Tensor
+
+Create a view of an existing `torch.Tensor` :attr:`input` with specified
+:attr:`size`, :attr:`stride` and :attr:`storage_offset`.
+
+.. warning::
+    :func:`as_strided` manipulates the internal data structure of tensors.
+    If done incorrectly, it can lead to the access of invalid memory and
+    result in garbage data.
+
+    More than one element of a created tensor may refer to a single memory
+    location. As a result, in-place operations (especially ones that are
+    vectorized) may result in incorrect behavior. If you need to write to
+    the tensors, please clone them first.
+
+    You should only use this function if you know what you're doing. A lot
```

Comment:
> You should only use this function if you know what you're doing.

I know I suggested this, but this sounds bad. Something like the following would be better:

"Many PyTorch functions that return a view on a tensor are internally implemented with this function. Those functions, like torch.expand, are easier to read and are therefore more advisable to use"